test/e2e/specs/editor/blocks/post-title.spec.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Post Title block', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'Can edit the post title', async ( { editor, page } ) => {
+		// Create a block with some text that will trigger a list creation.
+		await editor.insertBlock( { name: 'core/post-title' } );
+
+		// Add the post title
+		await page.locator( 'role=textbox[name="Add title"i]' ).focus();
+		await page.keyboard.type( 'Just tweaking the post title' );
```

Comment:
Nit: let's combine these together to:

```suggestion
		await page.locator( 'role=textbox[name="Add title"i]' ).focus();
		await page.keyboard.type(
			'role=textbox[name="Add title"i]',
			'Just tweaking the post title'
		);
```

Reply:
@kevin940726 

```
await page.keyboard.type(
    'role=textbox[name="Add title"i]',
    'Just tweaking the post title'
);
```

will actually type `'role=textbox[name="Add title"i]'` into the field, instead of the text we want.

Reply:
Ahh right, it should be `page.type` instead:

```suggestion
		await page.type(
			'role=textbox[name="Add title"i]',
			'Just tweaking the post title'
		);
```